README.md

```diff
@@ -51,57 +51,32 @@ After you're finished please remove all the comments and instructions!
   - [Run tests](#run-tests)
   - [Deployment](#triangular_flag_on_post-deployment)
 - [👥 Authors](#authors)
-- [🔭 Future Features](#future-features)
-- [🤝 Contributing](#contributing)
-- [⭐️ Show your support](#support)
-- [🙏 Acknowledgements](#acknowledgements)
-- [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 [Awesome Books] <a name="about-project"></a>
 
-> Describe your project in 1 or 2 sentences.
+> Books Library
 
-**[your_project__name]** is a...
+**[Awesome books]** is a...
 
 ## 🛠 Built With <a name="built-with"></a>
 
 ### Tech Stack <a name="tech-stack"></a>
 
 > Describe the tech stack and include only the relevant sections that apply to your project.
```

Comment:
- [ ] I noticed that there are placeholders associated to some sections/headers. The placeholder are present in the REAME template to help you with what to add in that respective section. They are meant to act as a guidance for you and not for your reader 😃 Please consider removing the placeholders from all sections 🎯 I know you can do that 💪
- [ ] Also, while adding the **_Future Features_** section, please make sure of adding 2-3 relevant future features related to you project 😄